code/modules/mob/mob.dm

```diff
@@ -1320,3 +1320,12 @@
 
 /mob/proc/get_target_zone()
 	return zone_sel?.selecting
+
+/mob/proc/is_on_special_ability_cooldown()
+	return FALSE
+
+/mob/proc/set_special_ability_cooldown(var/amt)
+	return
+
+/mob/proc/get_time_until_next_special_ability()
```

Comment:
I think this is a holdover from before you renamed it to `get_seconds_until_next_special_ability_string`. Since that's defined on `/mob/living` this override can probably be deleted?